src/index.rst

```diff
@@ -54,10 +54,10 @@ team and other Nextstrain users provide assistance.  For private inquiries,
     Installing <install>
     tutorials/running-a-workflow
     tutorials/creating-a-workflow
-    Exploring SARS-CoV-2 evolution <https://docs.nextstrain.org/projects/ncov/en/latest/index.html>
+    Exploring SARS-CoV-2 evolution <https://docs.nextstrain.org/projects/ncov/page/index.html>
```

Comment:
Didn't know about the `/page/` redirects. Thanks for sharing it here! This would be good to add to the "How-to tips" section of the README.

Also for the description of c43ef7fd7f8d79fe104c1aaebd88fc3d5eedd56b, I think you meant `…/page/…` instead of `…/pages/…`, but no big deal.

Reply:
Ah, good catch! Fixed in 8287793:

![image](https://user-images.githubusercontent.com/79913/215860595-13f600d1-036d-4771-918a-6fe663a01da3.png)


> This would be good to add to the "How-to tips" section of the README.

I agree it'd be good to note this somewhere… but the README feels enough like a sprawling mess (to me) right now that I don't want to touch it. Instead, I've added it to our fledgling style guide in 9fc90c5.